packages/alea-frontend/pages/api/resourceaccess-utils/resource-common-utils.ts

```diff
@@ -0,0 +1,14 @@
+import { RESOURCE_CACHE } from './resource-store';
+export function getCacheKey(resourceId: string, actionId: string) {
+  return `${resourceId}-${actionId}`;
```

Comment:
prefix key with a string that indicates that this is for resource assignments (and not ACL memberships):
 `resource-assignment:${resourceId}-${actionId}`
 
 Also add prefix to the ACL membership cache.